modules/View.js

```diff
@@ -1479,8 +1479,12 @@
      * @return {Array<String>} The fully expanded subattribute names
      */
     __generateSubAttributes: function(attr, model) {
-      var i, attrName, remainder, subAttrs, values,
-        firstBracket = attr.indexOf('[]');
+      var attrName;
```

Comment:
i feel like this took a backstep in readability

Reply:
Use
```
var remainder, subAttrs, values, indexes,
    firstBracket = attr.indexOf('[]');
```
you mean?

Reply:
yea. admittedly, im very much in the mindset of es6 now that doesn't have to hoist. but, having them all on one line is my least favorite